manifest/spark-operator.yaml

```diff
@@ -45,4 +45,5 @@ spec:
         imagePullPolicy: Always
         command: ["/usr/bin/spark-operator"]
         args:
-        - -logtostderr
\ No newline at end of file
+        - -logtostderr
+        - -default-spark-image=gcr.io/spark-operator/spark:v2.3.1
```

Comment:
@liyinan926 should I add this to all manifests as default, and rename this yaml file as `spark-operator-defaults` ?

Reply:
Additionaly, this image needs to exists with new change to use this flag - use of this flag with an old spark-operator version gives error during submission

Reply:
Essentially users need to first upgrade the operator before they can start leaving `image` empty.